### PR TITLE
remembrances & mausoleums

### PR DESCRIPTION
**Pull request by thomasgraham18**
added and linked all remembrances as well as mausoleums. all rewards also checkbox linked

### CODE REVIEW

File: data/rememberances.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+title: Remembrances & Mausoleums
+id: remembrances_mausoleums
+sections:
+  -
+    title: "Remembrances"
+    id: 1
+    table: ["Name", "Reward #1", "Reward #2", "Worth", "Obtained From"]
+    items:
+      - [1, "<a href=\"https://eldenring.wiki.fextralife.com/Remembrance+of+the+Grafted\">Remembrance Of The Grafted</a>", "Weapon - Axe Of Godrick", "Weapon - Grafted Dragon", "20,000 Runes", "Defeat Godrick The Grafted"]
```
Comment (azy2) [inline data/rememberances.yaml:9]: I think we want 2 checkboxes for each Remembrance. So people can track which choice they made. Use subheadings, something like this:
```
- "<a href=\"https://eldenring.wiki.fextralife.com/Remembrance+of+the+Grafted\">Remembrance Of The Grafted</a> (20,000 Runes)"
- [1, "Axe Of Godrick"]
- [2, "Grafted Dragon"]
```

File: data/rememberances.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      - [12, "<a href=\"https://eldenring.wiki.fextralife.com/Remembrance+of+the+Lichdragon\">Remembrance Of The Lichdragon</a>", "Incantation Spell - Fortissax's Lightning Spear", "Incantation Spell - Death Lightning", "30,000 Runes", "Defeat Lichdragon Fortissax"]
+      - [13, "<a href=\"https://eldenring.wiki.fextralife.com/Elden+Remembrance\">Elden Remembrance</a>", "Weapon - Marika's Hammer", "Weapon - Sacred Relic Sword", "50,000 Runes", "Defeat Elden Beast"]
+      - [14, "<a href=\"https://eldenring.wiki.fextralife.com/Remembrance+of+the+Rot+Goddess\">Remembrance Of The Rot Goddess</a>", "Weapon - Hand Of Malenia", "Incantation Spell - Scarlet Aeonia", "50,000 Runes", "Defeat Malenia, Blade Of Miquella"]
+      - [15, "<a href=\"https://eldenring.wiki.fextralife.com/Remembrance+of+the+Blood+Lord\">Remembrance Of The Blood Lord</a>", "Weapon - Mohgwyn's Sacred Spear", "Incantation Spell - Bloodboon", "30,000 Runes", "Defeat Mohgwyn, Lord Of Blood"]
+  -
+    title: "Walking Mausoleums"
+    id: 2
+    table: ["Region", "Location"]
+    items:
+      - [1, "Weeping Peninsula", "In the western area past the Church of Pilgrimage."]
```
Comment (azy2) [inline data/rememberances.yaml:29]: These look good but maybe turn it into a list instead of a table. Just describe the region in the sentence.

**Review by azy2 — CHANGES_REQUESTED** [review]

File: data/rememberances.yaml (hunk 2 above)
Comment (thomasgraham18) [reply data/rememberances.yaml]: Requested changes applied. Sorry for the wait.

**Review by azy2 — APPROVED** [review]
Looks great thank you

**Pull request merged**